copier/main.py

```diff
@@ -203,6 +206,7 @@ class Worker:
     unsafe: bool = False
     skip_answered: bool = False
     skip_tasks: bool = False
+    operation: Operation = "copy"
```

Comment:
thought: I find this a bit counter-intuitive. We have to pass an operation now to the worker. The operation could be something like "update" for example. But then we can `run_copy()` or `run_update()`.

It seems like duplicated features and inconsistent API behavior. If this is the case now, I think we should have a unified `run_operation()` method that runs either one or the other behind the scenes.

When the workers are created, they should be with the correct operation, and from there onwards use that unless `replace()` changes it.

Reply:
> We have to pass an operation now to the worker.

We mustn't pass it to the worker though, it's an internal reference that has to be `copy` by default, only overridden by calls to `replace` in `run_update`.

I wanted to declare this field as `init=False` for that reason, but that does not work together with `replace` by default (see the last paragraph on [dataclasses.replace](https://docs.python.org/3/library/dataclasses.html#dataclasses.replace)). This can be fixed by using a custom `replace` function though.